Cargo.toml

```diff
@@ -4,4 +4,9 @@ version = "0.1.4"
 edition = "2021"
 
 [dependencies]
+mint = { version = "0.5", optional = true }
 red4rs = { git = "https://github.com/jac3km4/red4rs", rev = "v0.1.4" }
+
+[features]
+default = ["mint"]
```

Comment:
I think it shouldn't be enabled by default, seems too highly specific of a thing to pull by default

Reply:
fixed in cabfa5d